apps/api/v2/src/modules/auth/guards/organizations/is-org.guard.ts

```diff
@@ -0,0 +1,28 @@
+import { OrganizationsRepository } from "@/modules/organizations/organizations.repository";
+import { Injectable, CanActivate, ExecutionContext, ForbiddenException } from "@nestjs/common";
+import { Request } from "express";
+
+import { Team } from "@calcom/prisma/client";
+
+@Injectable()
+export class isOrgGuard implements CanActivate {
```

Comment:
Was this supposed by be camel-cased like this?

Reply:
my bad will fixup